src/model/guild/audit_log.rs

```diff
@@ -454,7 +454,7 @@ impl<'de> Deserialize<'de> for AuditLogs {
                             users = Some(map.next_value::<Vec<User>>()?);
                         }
                         Ok(None) => break, // No more keys
-                        Err(e) => if e.description().contains("unknown field") {
+                        Err(e) => if format!("{}", e).contains("unknown field") {
```

Comment:
nitpick:
```suggestion
                        Err(e) => if e.to_string().contains("unknown field") {
```
Implementors of `Display` get a free `ToString` implementation that is equivalent to the `format!` macro here.